### ISSUE TITLE
Prettier v2.0.5 breaks linting

### ISSUE THREAD

**Issue by jonahsnider** [xojs/xo#466]
Nearly identical stacktrace to #441. Makes me think something is wrong in how XO uses Prettier.

```js
/home/runner/work/nextjs-starter/nextjs-starter/node_modules/.bin/xo
Error: Failed to load plugin 'prettier' declared in 'CLIOptions': Cannot find module 'eslint-plugin-prettier'
Require stack:
- /home/runner/work/nextjs-starter/nextjs-starter/__placeholder__.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:980:15)
    at Function.resolve (internal/modules/cjs/helpers.js:83:19)
    at Object.resolve (/home/runner/work/nextjs-starter/nextjs-starter/node_modules/eslint/lib/shared/relative-module-resolver.js:44:50)
    at ConfigArrayFactory._loadPlugin (/home/runner/work/nextjs-starter/nextjs-starter/node_modules/eslint/lib/cli-engine/config-array-factory.js:959:39)
    at /home/runner/work/nextjs-starter/nextjs-starter/node_modules/eslint/lib/cli-engine/config-array-factory.js:848:33
    at Array.reduce (<anonymous>)
    at ConfigArrayFactory._loadPlugins (/home/runner/work/nextjs-starter/nextjs-starter/node_modules/eslint/lib/cli-engine/config-array-factory.js:844:22)
    at ConfigArrayFactory._normalizeObjectConfigDataBody (/home/runner/work/nextjs-starter/nextjs-starter/node_modules/eslint/lib/cli-engine/config-array-factory.js:667:32)
    at _normalizeObjectConfigDataBody.next (<anonymous>)
    at ConfigArrayFactory._normalizeObjectConfigData (/home/runner/work/nextjs-starter/nextjs-starter/node_modules/eslint/lib/cli-engine/config-array-factory.js:596:20)
```

**Comment by sindresorhus** [comment]
Are you using the latest XO version? XO only recently (https://github.com/xojs/xo/commit/e05efc3ad8dd5b4694929f868b53276a774bb27d) received support for Prettier 2.

// @pvdlg

**Comment by pizzafox** [comment]
I am using the latest XO version. Prettier 2.0.4 works fine with XO.

**Comment by sindresorhus** [comment]
I've locked Prettier at 2.0.4 until we can resolve this: https://github.com/xojs/xo/releases/tag/v0.30.0

**Comment by BarryThePenguin** [comment]
It's unfortunate as there are fixes for using prettier in editor plugins, including [atom-linter-xo](https://github.com/xojs/atom-linter-xo), in prettier 2.0.5

https://github.com/prettier/prettier/issues/8047

Are they related at all?

**Comment by Gaspacchio** [comment]
We're on prettier `v2.1.1` and I've got the exact same problem. Is thers any fix coming?

As a temporary workaround, you can `add eslint-plugin-prettier` to your dependencies and use the latest version of prettier.

**Comment by pizzafox** [comment]
> This is almost certainly the same root cause as why having a different Prettier version breaks. However XO is importing or using other modules isn't resilient to version changes and seems to force you to use whatever version is specified in the XO `package.json`.

Same issue with XO having different version than you. #483

**Comment by papb** [comment]
@sindresorhus I believe this issue shouldn't have been closed. PR #505 contained the sentence `doesn't fix #466` but GitHub looked at `fix #466` and went out to close this.

**Comment by fregante** [comment]
Essentially same as https://github.com/xojs/xo/issues/546#issuecomment-2226958054.

It's an npm issue. Do a fresh `npm install` or wait for flat config.
